frontera.md

```diff
@@ -12,25 +12,70 @@ Then execute:
 
 ```
 module load tacc-apptainer
-singularity pull -F docker://seissol/training:latest
-singularity build -f my-training.sif singularity.def
-singularity run my-training.sif
+apptainer pull -F docker://seissol/training:latest
+apptainer build -f my-training.sif singularity.def
+apptainer run my-training.sif
+```
+
+You can also use the automatically generated container after pulling the docker container 
+
+```
+module load tacc-apptainer
+apptainer pull -F docker://seissol/training:latest
+apptainer run training_latest.sif
 ```
 
 You can abort the jupyter lab with Ctrl-C, confirm with `y`.
 Now you should see a directory `seissol-training`.
 This folder should contain four directories for different scenarios.
 
+You can also open the jupyter notebook that runs on Frontera on your local machine with the following steps:
+
+Step 1: change `SHARED_PATH="/your/path/to/container/"` in line 75 of `job.jupyter` to the path where your sigularity container is built.
+Step 2: Run
+```
+sbatch job.jupyter
+```
+Step 3: Check the job status with
+```
+squeue -u $USER
+```
+Step 4: Once the status changes from `PD` to `R`, you will find the job output in a generated file `jupyter.out`.
+Step 5: Check the last few lines with
+```
+tail jupyter.out
```

Comment:
`tail -f jupyter.out`
until you get ...  
(and then Ctrl+C)
?

Reply:
I think we may need to keep the job running. I change to the following descriptions (maybe this makes it clearer?):

Step 5: Check the last few lines with
```
tail jupyter.out
```
Wait a few seconds until you get in `jupyter.out` something like:
```
TACC: got login node jupyter port 60320
TACC: created reverse ports on Frontera logins
TACC: Your jupyter notebook server is now running at https://frontera.tacc.utexas.edu:60320/?token=2e0fade1f8b1ce00b303a7e97dd962c5cd10c17f03a245e8c761ca7e1d5e1597
```

Reply:
I mean, 
the  -f option of tail is a flag that tells tail to follow the file, meaning that it will continue to display new lines as they are added to the file.

Reply:
Uh, I see. I will change that! Thx!